ajax/libs/brython/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "brython",
+  "filename": "brython.js",
+  "version": "3.3.2",
+  "description": "Python 3 in the browser",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brython-dev/brython.git"
+  },
+  "keywords": [
+    "Python",
+    "browser"
+  ],
+  "author": "Pierre Quentel",
+  "license": "BSD-2-Clause",
```

Comment:
@x09326 It shows that it's `BSD-3-Clause`(https://github.com/brython-dev/brython/blob/3.2.7/LICENCE.txt) on GitHub repo, but it shows `BSD-2-Clause` on NPM. Please confirm it with author, thanks.